src/types.cpp

```diff
@@ -0,0 +1,47 @@
+#include "create/types.h"
+
+namespace create {
+  RobotModel::RobotModel(const std::string& name) {
```

Comment:
In order to prevent invalid strings being passed into the constructor, I was suggesting to make it a copy constructor instead. 

But, as it stands we only expect the three robot models defined on [lines 44-46](https://github.com/AutonomyLab/libcreate/pull/25/files#diff-5008d1e9054cf71e83b08026b93e3ac1R44) to be used. Therefore, I think we can safely remove this constructor altogether.